### PR TITLE
Fix jsp.feature.not.loaded.correctly CWWJS0001E ID and message

### LINKED ISSUE #22707: Fix jsp.feature.not.loaded.correctly ID message
POC Item 97

CWWJS0001 needs an `E` at the end.
`jsp.feature.not.loaded.correctly=CWWJS0001: The JSP [Pages] feature did not load with an acceptable version.
`

Wording could also be improved

### PR DESCRIPTION
**Pull request by volosied**
fixes #22707

Update text per Fred Rowe's suggestion.

### CODE REVIEW

**Comment by volosied** [comment]
#libby

**Comment by volosied** [comment]
@pnicolucci  Should we update the messsages for websocket & webcontainer to be consistent? 

https://github.com/OpenLiberty/open-liberty/blob/21f161e065a064ce64a4dc384511bb2e620b6559/dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops#L229 

https://github.com/OpenLiberty/open-liberty/blob/2fd4a880754c37a988c5ed9ac4f1ea5988e465d6/dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops#L1562

File: dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops [hunk 1]
```diff
@@ -1259,7 +1259,7 @@ system.java.compiler.not.found=JSPG0304W: The JSP attribute, useJDKCompiler, is
 system.java.compiler.not.found.explanation=The JSP container was unable to find the system compiler, which usually indicates that the Java SDK jar file, tools.jar, was not found in the class path.
 system.java.compiler.not.found.useraction=Ensure that the Java SDK jar file, tools.jar, is in the class path and verify that the Java SDK supports the Java Compiler API. See Java Specification Request 199 for more information.
 
-jsp.feature.not.loaded.correctly=CWWJS0001: The JSP [Pages] feature did not load with an acceptable version.
+jsp.feature.not.loaded.correctly=CWWJS0001E: An acceptable version of the JSP [Pages] feature was not loaded.
```
Comment (dmuelle) [inline dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops]: Do the square brackets around [Pages] serve a programmatic function?

Comment (pnicolucci) [reply dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops]: We're trying to list the new name for the specification. It used to be JSP, for example, JSP 2.2 or JSP 2.3 but starting with Jakarta EE9 it is Pages, for instance, Pages 3.0 or Pages 3.1.

Comment (pnicolucci) [reply dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops]: Would JSP (Pages) be better? Other suggestions?

Comment (dmuelle) [reply dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops]: Do we need to maintain the old spec names in the messages? The easiest solution would be to just use the new names (same for Java/Jakarta). If that's likely to cause confusion then I guess parentheses are better than brackets. Another possibility is

`The Pages feature (formerly JSP) did not load with an acceptable version.`

Comment (volosied) [reply dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops]: I've made that update along with a few others. Please review again, @dmuelle. Thanks!

File: dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops [hunk 2]
```diff
@@ -1259,9 +1259,9 @@ system.java.compiler.not.found=JSPG0304W: The JSP attribute, useJDKCompiler, is
...
 system.java.compiler.not.found.useraction=Ensure that the Java SDK jar file, tools.jar, is in the class path and verify that the Java SDK supports the Java Compiler API. See Java Specification Request 199 for more information.
 
-jsp.feature.not.loaded.correctly=CWWJS0001: The JSP [Pages] feature did not load with an acceptable version.
-jsp.feature.not.loaded.correctly.explanation=The JSP [Pages] feature is loaded with a version that corresponds to the Java [Jakarta] EE specification of that JSP [Pages] feature. The feature was loaded with no version defined.
-jsp.feature.not.loaded.correctly.useraction=Check the server.xml file for the existence of multiple JSP [Pages] features. The file must specify no more than one Pages feature.
+jsp.feature.not.loaded.correctly=CWWJS0001E: The Pages feature (formerly JSP) did not load with an acceptable version.
+jsp.feature.not.loaded.correctly.explanation=The feature loaded without an acceptable version. The application may behave differently than expected.
```
Comment (dmuelle) [inline dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops]:
```suggestion
jsp.feature.not.loaded.correctly.explanation=The feature loaded without an acceptable version. The application might behave differently than expected.
```

**Comment by posmith** [comment]
L2 message review completed

**Comment by volosied** [comment]
#build

**Review by dmuelle — APPROVED** [review]
LGTM thanks

**Comment by volosied** [comment]
Some GUI test ( com.ibm.ws.ui.tool.explore.fat.gui.nd.rw.tests.ListViewTest ) failed, but otherwise build was good.

**Comment by pnicolucci** [comment]
> @pnicolucci Should we update the messsages for websocket & webcontainer to be consistent?
> 
> https://github.com/OpenLiberty/open-liberty/blob/21f161e065a064ce64a4dc384511bb2e620b6559/dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops#L229
> 
> https://github.com/OpenLiberty/open-liberty/blob/2fd4a880754c37a988c5ed9ac4f1ea5988e465d6/dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops#L1562

Yes, I think we should be consistent in our messaging, can you get an issue opened up and associated with the issue for this PR so we don't forget to clean these up?  Thanks!

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#libby

**Pull request merged**